include/FreeRTOS.h

```diff
@@ -896,6 +896,11 @@
     #define configSUPPORT_DYNAMIC_ALLOCATION    1
 #endif
 
+#ifndef configSUPPORT_SEPARATE_STACK_ALLOCATION

```

Comment:
Please change the config parameter name to `configSTACK_ALLOCATION_FROM_SEPARATE_HEAP`.